features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/DefinitionOutdated.jsx

```diff
@@ -116,7 +119,7 @@ export default function DefinitionOutdated(props) {
                                 defaultMessage='Cancel'
                             />
                         </Button>
-                        <a
+                        {/* <a
```

Comment:
Can't we remove this commented code section ?

Reply:
Fixed